kmcd-prelude.md

```diff
@@ -102,9 +102,9 @@ module KMCD-PRELUDE
 
          // Initialize Spot for gold
          transact ADMIN Spot . init     "gold"
-         transact ADMIN Spot . setPrice "gold" 3 ether
-         transact ADMIN Spot . file       mat "gold" 1
-         transact ADMIN Spot . file       par 1
+         transact ADMIN Spot . setPrice "gold" Int2Wad(3 ether)
```

Comment:
This is going to do a weird thing when we switch to `Int`. I guess it's okay, we can fix it then (with the `Int2Wad` invocation, it's more in line with the intent to just change `3 ether` to `3`).

Reply:
Would be nice to be able to e.g. write `3 * 1Wad`.

Reply:
Later it will just be `wad(3 ether)`. But for now I'm using the `Int2Wad` converter to keep the sort inferencer happy.

Reply:
I've gone ahead and included the change that puts `wad(...)`, `ray(...)`, `rad(...)` instead of the `Int2*` functions.